src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/ZLibNative.cs

```diff
@@ -312,7 +312,7 @@ public ErrorCode InflateEnd()
             }
 
             // This can work even after XxflateEnd().
-            public string GetErrorMessage() => _zStream.msg != ZNullPtr ? Marshal.PtrToStringAnsi(_zStream.msg) : string.Empty;
+            public string? GetErrorMessage() => _zStream.msg != ZNullPtr ? Marshal.PtrToStringAnsi(_zStream.msg) : string.Empty;
```

Comment:
In what situation does this return null?

Reply:
Marshal.PtrToStringAnsi(...) returns nullable

Reply:
> Marshal.PtrToStringAnsi(...) returns nullable

Not in the way it's used here.  It'll return null if the pointer is null, but the call is only being used here when it's not.